root/etc/cont-init.d/45-nginx

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/with-contenv bash
+
+# copy default config files if they don't exist
+if [[ ! -f /config/nginx/http-confs/unauthorized.conf ]]; then
+    cp /defaults/nginx/http-confs/unauthorized.conf.sample /config/nginx/http-confs/unauthorized.conf
+fi
+if [[ ! -f /config/nginx/location-confs/proxy.conf ]]; then
+    cp /defaults/nginx/location-confs/proxy.conf.sample /config/nginx/location-confs/proxy.conf
+fi
+if [[ ! -f /config/nginx/server-confs/502.conf ]]; then
+    cp /defaults/nginx/server-confs/502.conf.sample /config/nginx/server-confs/502.conf
+fi
+if [[ ! -f /config/nginx/server-confs/ssl.conf ]]; then
+    cp /defaults/nginx/server-confs/ssl.conf.sample /config/nginx/server-confs/ssl.conf
+fi
+if [[ ! -f /config/www/502.html ]]; then
+    cp /defaults/www/502.html /config/www/502.html
+fi
+
+# patch authelia-server.conf for CVE-2021-32637
+if [[ -f /config/nginx/server-confs/authelia-server.conf ]]; then
+    if ! grep -q 'if ($request_uri ~' /config/nginx/server-confs/authelia-server.conf; then
+        sed -i '/internal;/a \ \ \ \ if ($request_uri ~ [^a-zA-Z0-9_+-=\\!@$%&*?~.:#'\''\\;\\(\\)\\[\\]]) { return 401; }' /config/nginx/server-confs/authelia-server.conf
+    fi
+fi
```

Comment:
we can get rid of this patch as well, been over a year and is mitigated by Authelia upstream